Project.toml

```diff
@@ -31,8 +31,9 @@ julia = "^1"
 Cbc = "9961bab8-2fa3-5c5a-9d89-47fab24efd76"
 Ipopt = "b6b21f68-93f8-5de0-b562-5493be1d77c9"
 Juniper = "2ddba703-00a4-53a7-87a5-e8b9971dde84"
+Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
```

Comment:
I am ok with the Juniper logs going into the terminal, and I would not want to miss any new warnings from other packages (e.g. JuMP/MOI) while running tests.

So we can remove this dependency in favor of the default behavior.

Reply:
Fine with me! I will change this.